src/components/Table/Header.tsx

```diff
@@ -0,0 +1,60 @@
+import React, { ReactElement } from 'react'
+import { HeaderGroup, CellValue } from 'react-table'
+import classNames from 'classnames'
+import { Icon, Text } from '../'
+import { calcColumnWidth, CHECKBOX_HEADER_ID } from './utils'
+
+type Props = {
+  fixedHeader: boolean
+  tableWidth: number
+  headerGroup: HeaderGroup
+}
+
+function Header({ headerGroup, tableWidth, fixedHeader = false }: Props): ReactElement {
+  return (
+    <tr
+      {...headerGroup.getHeaderGroupProps()}
+      className={classNames({ fixed_header: fixedHeader })}
+    >
+      {headerGroup.headers.map((column: CellValue, i: number, arr: CellValue[]) => {
+        const isSelection = column.id === CHECKBOX_HEADER_ID
+
+        const style = {
+          width: isSelection ? 17 : calcColumnWidth(column.widthInPercent, tableWidth),
+          ...(!isSelection && column.minWidth ? { minWidth: column.minWidth } : {}),
+          ...(!isSelection && column.maxWidth ? { maxWidth: column.maxWidth } : {}),
+          ...(!isSelection && column.width ? { width: column.width } : {})
+        }
+
+        return (
+          <th
+            key={i}
+            {...column.getHeaderProps(
+              column?.columnProps?.sortable ? column.getSortByToggleProps() : undefined
+            )}
+            className={classNames({
+              fixed_column_left: column?.fixed === 'left',
+              fixed_column_right: column?.fixed === 'right',
+              fixed_checkbox_header:
+                column?.id === CHECKBOX_HEADER_ID && arr[i + 1]?.fixed === 'left'
```

Comment:
`column?.id === CHECKBOX_HEADER_ID` instead of this use `isSelection ` constant.